.ci/compute_projects.py

```diff
@@ -216,16 +227,16 @@ def get_env_variables(modified_files: list[str], platform: str) -> Set[str]:
     projects_to_test = _compute_projects_to_test(modified_projects, platform)
     projects_to_build = _compute_projects_to_build(projects_to_test)
     projects_check_targets = _compute_project_check_targets(projects_to_test)
-    runtimes_to_test = _compute_runtimes_to_test(projects_to_test)
-    runtimes_check_targets = _compute_runtime_check_targets(runtimes_to_test)
+    runtimes_to_build = _compute_runtimes_to_test(projects_to_test)
```

Comment:
It looks like "_compute_runtimes_to_test" is actually computing the runtimes to build, not to test.  It might be good to rename the method (and its local variables) to reflect this, so future maintainers don't get confused by this...(If I've misunderstood something, let me know)

Reply:
Ah, ok thanks!